src/global-styles.scss

```diff
@@ -1,14 +1,71 @@
-$example-color: cornflowerblue;
+// gray-700
+// $bg-color: #374151;
 
-.example-class {
-    color: $example-color;
+// zinc-900
+$bg-color: #18181b;
+$text-color: #f8fafc;
+
+// gray-800
+// $card-lv-1-color: #1f2937;
+
+// zinc-700
+$card-lv-1-color: #3f3f46;
+
+
+
+html {
+    background-color: $bg-color;
+    color: $text-color;
+    height: 100%;
+}
+
+body {
+    height: 100%;
+}
+
+.sveltekit-container {
+    height: 100%;
 }
 
 .button {
+    all: unset;
     padding: 8px;
     border-radius: 4px;
     border: 2px solid #999;
     text-decoration: none;
     cursor: pointer;
+    user-select: none;
     display: inline-block;
-}
\ No newline at end of file
+}
+
+a, a:visited, a:hover, a:active {
```

Comment:
This is doing nothing.

Reply:
If not for this, my button-which-is-secretly-an-a-tag will have its text change color for a split second when you click it.